contractdata/FACILITY/_FACILITY_CHALLENGES.json

```diff
@@ -28,6 +29,7 @@
                     "ParentLocationId": "LOCATION_PARENT_ICA_FACILITY",
                     "Type": "contract",
                     "DifficultyLevels": [],
+                    "OrderIndex": 10000,
```

Comment:
This goes for all challenge JSON files now with this property:
Do we need to do anything with this? Or will the game sort it out for us?

Reply:
I guess it has to do with the ordering of the challenges and the game probably won't sort it out automatically.